src/js/common/schemaform/validation.js

```diff
@@ -79,51 +81,89 @@ export function transformErrors(errors, uiSchema) {
  */
 
 export function uiSchemaValidate(errors, uiSchema, formData, formContext, path = '') {
-  const currentData = _.get(path, formData);
-  if (uiSchema.items && currentData) {
-    currentData.forEach((item, index) => {
-      const newPath = `${path}[${index}]`;
-      if (!_.get(newPath, errors)) {
-        _.get(path, errors)[index] = {
-          __errors: [],
-          addError(error) {
-            this.__errors.push(error);
-          }
-        };
-      }
-      uiSchemaValidate(errors, uiSchema.items, formData, formContext, newPath);
-    });
-  } else {
-    Object.keys(uiSchema)
-      .filter(prop => !prop.startsWith('ui:'))
-      .forEach((item) => {
-        const nextPath = path !== '' ? `${path}.${item}` : item;
-        if (!_.get(nextPath, errors)) {
-          _.get(path, errors)[item] = {
+  if (uiSchema) {
```

Comment:
It happens the very first time this function is called, when we're at the root of the form schema.